mkdocs_git_revision_date_localized_plugin/util.py

```diff
@@ -183,7 +185,7 @@ def add_spans(date_formats: Dict[str, str]) -> Dict[str, str]:
         """
         for date_type, date_string in date_formats.items():
             date_formats[date_type] = (
-                '<span class="git-revision-date-localized-plugin git-revision-date-localized-plugin-%s">%s</span>'
+                Markup('<span class="git-revision-date-localized-plugin git-revision-date-localized-plugin-%s">%s</span>')
                 % (date_type, date_string)
```

Comment:
Sorry about the bad change.

I tried out the plugin only in the default config without overrides, and it kept working.
On the linked site, a non-default style is probably used, that's why it's observed there.

My mistake is as follows:

I assumed that the `date_string` string is text, not HTML. I intentionally added `Markup` in a way that would escape that string. However there is one type of date_string that *is* HTML, it's here:
https://github.com/timvink/mkdocs-git-revision-date-localized-plugin/blob/fe156ff64f2a5ffcea95f7c2a3f92eae4de8bb85/mkdocs_git_revision_date_localized_plugin/dates.py#L44


I could've added `Markup` in a simpler and more careful way and should've just done that.

Bad:

```python
                Markup('<span class="git-revision-date-localized-plugin git-revision-date-localized-plugin-%s">%s</span>')
                % (date_type, date_string)
```

Good:

```python
             Markup(
                '<span class="git-revision-date-localized-plugin git-revision-date-localized-plugin-%s">%s</span>')
                % (date_type, date_string)
             )
```